externallib.php

```diff
@@ -286,4 +286,55 @@ public static function set_self_enrolment_method_key_returns() {
             )
         );
     }
+
+    /**
+     * Returns description of method parameters
+     * @return external_function_parameters
+     */
+    public static function get_user_uuids_parameters() {
+        return new external_function_parameters(
+            array(
+                'user_ids' => new external_multiple_structure(
+                    new external_single_structure(
+                        array(
+                            'id' => new external_value(PARAM_INT, VALUE_OPTIONAL, 'user id'),
+                        )
+                    ), 'User IDs requested', VALUE_DEFAULT, array()
+                )
+            )
+        );
+    }
+
+    /**
+     * Get the uuids associated with the given user ids.
+     * @param array series of user ids or none (return all uuid mappings)
```

Comment:
There are PHPDoc errors on this. I realized we can actually add this to our CI checking so it's not a manual thing to check and created [this PR](https://github.com/openstax/moodle-local_raise/pull/3) in the `local_raise` plugin to test / confirm (that actually flagged existing linter errors which I'll fix there). We can / should add the same CI configuration change here.